components/backdrop/backdrop.js

```diff
@@ -49,15 +49,16 @@ class Backdrop extends LitElement {
 			:host([slow-transition]) {
 				transition: opacity 1200ms ease-in;
 			}
-			:host([_state=null][no-animate-hide]) {
```

Comment:
This is an example where I "corrected" the descending specificity of the selectors (which I did not ultimately enforce).